### ISSUE TITLE
After importing environments from folder, not getting listed in environments.

### ISSUE THREAD

**Issue by WSMathias** [a85/POSTMan-Chrome-Extension#4794]
**To Reproduce**
1. export environment collection to  folder myCollections 
2. delete that collection in the postman
3. select the **import folder** and choose myCollection folder. 
4. See message imported successfully
5. now expand the environment list, it will not be listed there.
6. now got to  **Manage environment** and import the same environment. 
7. see there will be copies of the same environment.

**Expected behavior**
After import from folder refresh environment dropdown list.

**Screenshots**

**App information (please complete the following information):**
 - App Type: Native App
 - Postman Version: 6.1.4
 - OS: macOS High Sierra 10.13.5

**Additional context**
It works as expected when importing from  **Manage environment**.
**It's annoying**.
Bad user experience.

**Comment by numaanashraf** [comment]
Thanks for reporting this @WSMathias. We are looking into this.

**Comment by kamalaknn** [comment]
Hi @WSMathias 

I was able to replicate this. This seems to be happening when there are multiple updates being made in parallel, in your case a collection import immediately followed by an environment import. In this scenario, the updates are accurately applied on storage but were processed out of order in view. That's why the view corrects itself after any action, or restart.

We've fixed this. We'll be pushing it out in the next release on stable channel. We'll push this out to Canary channel soon.

I'll keep this thread updated on the status. Thanks.
